girlsgoit/CSS-Workshop
# Update template styles and markup

Changes:
---
- everything flex
- change BEM style selectors to `-` only
- add portfolio section as bonus/extra work, hidden by default
- change tag selectors to classes: ex: 

How To Test:
---
- clone the branch
- include `example.css` in `index.html`
- check the end result
- to test the portfolio section remove `display: none;` rule from `style.css`
